src/components/ProfileBlock.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { HexString, Profile } from "../utilities/types";
+
+interface ProfileProps {
+  walletAddress: HexString | null;
+  socialAddress: HexString | null;
+  profile: Profile | null;
```

Comment:
do we want to include the `or null` in the types file instead of having to iterate it throughout?

Reply:
So like make some proper WalletAddress and SocialAddress types? Yeah sure. I can do that. I'll add HexAddress that can be either a HexString or null.

Reply:
I think the right way to do this is instead to simply not pass walletAddress and socialAddress unless they exist and we can make these params optional.